tests/test_ingest_external_file.rs

```diff
@@ -428,3 +428,26 @@ fn test_mem_sst_file_writer() {
         ],
     );
 }
+
+#[test]
+fn test_set_external_sst_file_global_seq_no() {
+    let db_path = TempDir::new("_rust_rocksdb_set_external_sst_file_global_seq_no_db").expect("");
+    let db = create_default_database(&db_path);
+    let path = TempDir::new("_rust_rocksdb_set_external_sst_file_global_seq_no").expect("");
+    let file = path.path().join("sst_file");
+    let sstfile_str = file.to_str().unwrap();
+    gen_sst(
+        ColumnFamilyOptions::new(),
+        Some(db.cf_handle("default").unwrap()),
+        sstfile_str,
+        &[(b"k1", b"v1"), (b"k2", b"v2")],
+    );
+
+    let handle = db.cf_handle("default").unwrap();
+    let seq_no = 0;
+    assert!(set_external_sst_file_global_seq_no(&db, &handle, sstfile_str, seq_no).is_ok());
```

Comment:
How to check the global_seq_no of this sst file is what you have set?

Reply:
BTW, you should cover two cases, 1) the original global sequence number is equal to what you set, 2) not equal

Reply:
If failed, there will be a bad status returned, the `assert` will fail.

Reply:
How do you know `set_external_sst_file_global_seq_no` act as you expect when it return OK.

Reply:
If not, in rust-rocksdb, there will be an error in `Result<u64, String>`.